config_example.py

```
@@ -0,0 +1,53 @@
+from pydantic_settings import BaseSettings, NoDecode
+
+
+class Settings(BaseSettings):
+    # ── Keycloak ──────────────────────────────────────────
+    keycloak_url: str
+    keycloak_realm: str
+    keycloak_client_id: str
+    keycloak_client_secret: str
+
+    # ── Приложение ────────────────────────────────────────
+    host: str = "0.0.0.0"
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)